test/e2e/es/mutation_test.go

```diff
@@ -227,6 +233,52 @@ func TestMutationWithLargerMaxUnavailable(t *testing.T) {
 	RunESMutation(t, b, mutated)
 }
 
+func TestMutationWhileLoadTesting(t *testing.T) {
+	b := elasticsearch.NewBuilder("test-while-load-testing").
+		WithESMasterDataNodes(3, elasticsearch.DefaultResources).
+		WithHTTPLoadBalancer()
```

Comment:
I missed the fact that the test is using a LoadBalancer, we should avoid that because this kind of service is only available on some K8S deployments.
For example I'm not sure that we will be able to provision a LoadBalancer IP on Openshift 3/4

Reply:
This was to allow running (and debugging) the test locally, but @barkbay pointed me offline to the fact we can do port forwarding instead. I'll make a fix.